src/diffusers/pipelines/aura_flow/pipeline_aura_flow.py

```diff
@@ -408,6 +428,7 @@ def __call__(
         max_sequence_length: int = 256,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
+        joint_attention_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
Let's rename this to `attention_kwargs`.

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/6514892fd53f06502329b3927714f7d166db89fd

Reply:
Copying from Flux would break this -- I'd prefer to keep as-is. Let me know if you'd still like to change it.

Reply:
Actually, done in https://github.com/huggingface/diffusers/pull/10216/commits/d6028cdc53d6d30c14022f8045f87c7f59c466c1.